conftest.py

```diff
@@ -22,6 +22,10 @@ def user():
 def staffuser():
     return UserFactory(short_name="Hello", password='password', is_staff=True)
 
+@pytest.fixture()
+def systemuser():
+    return UserFactory(serial='__system__', password='!!')
```

Comment:
This fixture was added in the "system user" pull request. You can just remove it from yours when you rebase.